app/controllers/messages_controller.rb

```diff
@@ -107,6 +107,13 @@ def outbox
     @title = t ".title"
   end
 
+  # Display the list of muted messages received by the user.
+  def muted
+    @title = t ".title"
+
+    redirect_to inbox_messages_path if current_user.muted_messages.none?
```

Comment:
The redirect is meant to handle the situation in which a user moves the last muted message back into the Inbox, like so:

![2023-10-26 20 40 42](https://github.com/openstreetmap/openstreetmap-website/assets/28648/16ca6c81-bca2-4f98-ac55-8f6033eed7f4)

Once there are no muted Messages anymore, the `Muted`-Tab is hidden and the user is redirected back to the Inbox.

Rendering the template instead would create an inconsistent UI because no Tab would be active. Or we would need to display the `Muted`-Tab even though there are no messages and then hide it once the user clicks on any other link.

Why would you prefer rendering the template?

Reply:
We also hide the `Muted users`-Tab on the Account-page, so I understand that we settled on this behaviour (for now) => resolving the conversation.